### ISSUE TITLE
Yale Smart Alarm - Integration 

### ISSUE THREAD

**Issue by dchill3333** [domwillcode/yale-smart-alarm-client#112]
# Problem/Motivation

> Home Assistant Yale Smart Alarm client now not working

## Expected behavior

> Able to access my Yale Smart Alarm via HA

## Actual behavior

>Logger: homeassistant.components.yale_smart_alarm
Source: helpers/update_coordinator.py:354
integration: Yale Smart Living ([documentation](https://www.home-assistant.io/integrations/yale_smart_alarm), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+yale_smart_alarm%22))
First occurred: 12 September 2024 at 20:22:51 (202 occurrences)
Last logged: 10:30:15

Unexpected error fetching yale_smart_alarm data
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 354, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/yale_smart_alarm/coordinator.py", line 58, in _async_update_data
    for device in updates["cycle"]["device_status"]:
                  ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
KeyError: 'device_status'

## Steps to reproduce

> (Unsure)

## Proposed changes

> (IN/A)
![Screenshot 2024-09-13 102016](https://github.com/user-attachments/assets/9975825c-2ca3-4bd5-bb0e-12c1f4cec89b)
![Screenshot 2024-09-13 103332](https://github.com/user-attachments/assets/06e77974-ef2c-4bfa-90d3-6c7ad0f8a7be)

**Comment by gjohansson-ST** [comment]
This is a HA core issue and should have been opened there and not here. 
But it's anyhow fixed in the next release (2024.9.2). 
Closing this here

**Comment by dchill3333** [comment]
Hi 

Many thanks for your very prompt reply and my apologies posting in the wrong area, a newbie mistake of mine. Have a great weekend

David
